controllers/tools/seed.js

```diff
@@ -0,0 +1,25 @@
+const uuidv4 = require('uuid/v4');
+const userService = require('../../services/user');
+
+function init({ router }) {
+    if (!process.env.USE_LOCAL_DATABASE) {
+        return;
+    }
+
+    router.post('/seed/user', (req, res) => {
```

Comment:
We could wrap this in an environment check too for safety so we never end up with a public route that creates a privileged user, but perhaps the `USE_LOCAL_DATABASE` flag check is enough?